src/schedule.py

```diff
@@ -38,18 +38,23 @@ def AddEvent(self, weekday, event):
 EVT_NEVER_ON = 3
 EVENT_TYPES = {EVT_FIXED: 'FIXED', EVT_SUNSET: 'SUNSET', EVT_NEVER_ON: 'NEVER-ON'}
 
-
-def get_civil_twilight(observer, event_date, observer_timezone):
-    ''' wrapper function for twilight sunset start'''
-    twilight_start_end = twilight(observer, event_date, SunDirection.SETTING, observer_timezone)
-    return twilight_start_end[0]
-
-def getEventType(schedule, event_date, observer, observer_timezone):
+class ScheduleObserver(Observer):
+    '''schedule implementation of astral.Observer to aggregate timezone'''
+    timezone: str = "UTC"
+    def __init__(self, timezone="UTC", **kwargs):

```

Comment:
General use is to pass in a tzinfo object like so: `tzinfo: datetime.tzinfo = pytz.utc` instead of strings.

Reply:
okay, see my most recent push. I like this better. I chose pytz over datetime as astral uses pytz timezones.